substreams/src/parser.rs

```diff
@@ -0,0 +1,53 @@
+use pest::{iterators::Pair, Parser};
+use pest_derive::Parser;
+
+#[derive(Parser)]
```

Comment:
Can you check if it's possible to generate the code and commit it? It would be preferable leading to faster compiling for users since less time running the derive macro.